src/js/components/Item_birthday.vue

```diff
@@ -0,0 +1,332 @@
+                        placeholder="Name"
+                        v-model="birthdayModal.name"
+                    >
+                </div>
+
+                <div class="newtab__modal_field">
+                    <input
+                        class="newtab__modal_input"
+                        type="text"
+                        placeholder="Date (01-01)"
+                        v-model="birthdayModal.date"
+                    >
+                </div>
+
+                <div class="newtab__modal_footer">
+                    <button
+                        class="newtab__modal_button"
+                        @click="submitBirthdayModal()"
+                        v-text="birthdayModal.modalName"
+                    ></button>
+                </div>
+            </div>
+        </modal>
+
+        <modal
+            name="importBirthdaysModal"
+            :width="435"
+            :height="380"
+        >
+            <div class="newtab__modal">
+                <div class="newtab__modal_header">
+                    <h1 class="newtab__modal_title" v-text="'Import'"></h1>
+
+                    <button class="newtab__modal_close" @click="closeImportBirthdayModal()">
+                        <span><i class="fas fa-times"></i></span>
+                    </button>
+                </div>
+
+                <div class="newtab__modal_field">
+                    <textarea
+                        class="newtab__modal_input"
+                        rows="9"
+                        placeholder="Paste birthday json"
+                        v-model="birthdayModal.import"
+                    ></textarea>
+                </div>
+
+                <div class="newtab__modal_footer">
+                    <button
+                        class="newtab__modal_button"
+                        @click="importBirthdays()"
+                        v-text="'Save'"
+                    >
+                    </button>
+                </div>
+            </div>
+        </modal>
+    </div>
+</template>
+
+<script>
+import moment from 'moment';
+
+export default {
+    props: {
+        items: {
+            type: Array,
+            required: true,
+        },
+
+        sectionKey: {
+            type: Number,
+            required: true,
+        },
+    },
+
+    data: () => ({
+        birthdayModal: {
+            itemKey: null,
+            name: '',
+            date: '',
+            modalName: '',
+            import: '',
+        },
+
+        birthdayMonthModal : [],
+
+        allBirthdayModal : [],
+    }),
+
+    methods: {
+        getMonth (birthdays) {
+            return birthdays.filter(birthday =>
+                Number(birthday.date.split('-')[1]) === moment().month() + 1,
+            );
+        },
+
+        getToday (birthdays) {
+            return birthdays.filter(birthday =>
+                Number(birthday.date.split('-')[0]) === moment().date() && Number(birthday.date.split('-')[1]) === moment().month() + 1,
```

Comment:
this validation:

```js
Number(birthday.date.split('-')[1]) === moment().month() + 1
```

is exactly the same of `getMonth()`, use `this.getMonth()` instead of `birthdays` variable